api/integrations/basecamp/zulip_basecamp_mirror

```diff
@@ -120,7 +121,11 @@ def run_mirror():
         since = since.string
     except (AssertionError, IOError) as e:
         logging.warn("Could not open resume file: %s" % (e.message or e.strerror,))
-        since = (datetime.utcnow() - timedelta(hours=config.BASECAMP_INITIAL_HISTORY_HOURS)).isoformat() + "-00:00"
+        since = (datetime.utcnow().replace(
+            tzinfo=pytz.utc
+        ) - timedelta(
+            hours=config.BASECAMP_INITIAL_HISTORY_HOURS
+        )).isoformat() + "-00:00"
     try:
```

Comment:
This is for logging in a certain format, so you can leave it as is (without the `tzinfo`)